package.json

```diff
@@ -9,20 +9,19 @@
     "url": "git@github.com:box/box-content-preview.git"
   },
   "devDependencies": {
+    "@babel/core": "7.0.0-beta.44",
```

Comment:
Are there any concerns with using a beta release?

Reply:
Yep, I suggest waiting for the official release